app/controllers/application_controller.rb

```diff
@@ -1,4 +1,11 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  before_action :set_subjects
+
+  private
+
+  def set_subjects
```

Comment:
Interesting, I guess rubocop doesn't check this but usually we indent anything below `private` to give a visual cue.